riscv-executor/src/lib.rs

```diff
@@ -379,13 +382,23 @@ mod builder {
         Submachine, SubmachineBoxed, PC_INITIAL_VAL,
     };
 
-    // TODO: where do we get this from? I don't think we want to depend on the linker crate just for this
-    const MIN_DEGREE: u32 = 1 << 5;
+    fn namespace_degree_range<F: FieldElement>(
```

Comment:
Wouldn't it be easier to read this from the PILGraph or some higher level data structure in Pipeline that could be passed here? Or is this safe enough?

Reply:
It should be the same in the end i think? Columns in a namespace should have the same degree no?

Reply:
yea I just meant the way to retrieve the numbers

Reply:
here we're already dealing in namespaces/columns and we already have the optimized PIL, seemed like the easiest way

Reply:
ok